<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_issued</class>
 <widget class="QMainWindow" name="book_issued">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1884</width>
    <height>872</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Isssued Book</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bookholics.qrc">
    <normaloff>:/image/1.png</normaloff>:/image/1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>881</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Member</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_member">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Borrow ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_borrow_id">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
        </property>
        <property name="placeholderText">
         <string>Enter valid ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>57</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>57</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="to_date">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>311</width>
       <height>183</height>
      </rect>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="from_date">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>321</width>
       <height>183</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>340</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ShowDetails</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="return_deatils">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>410</y>
      <width>881</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Book Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Book Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Book Author</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Book Edition</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Borrow Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Department</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Class</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Division</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="bookholics.qrc"/>
 </resources>
 <connections/>
</ui>
